openapi/data_repository_service.swagger.yaml

```diff
@@ -194,57 +181,85 @@ definitions:
           sha512
   Bundle:
     type: object
-    required: ['id', 'object_ids', 'created', 'updated', 'version', 'checksums']
     properties:
       id:
         type: string
-        description: |-
+        description: >-
           An identifier, unique to this Data Bundle
-      object_ids:
-        type: array
-        items:
-          type: string
+      name:
+        type: string
         description: |-
-          The list of Data Objects that this Data Bundle contains.
+          A string that can be optionally used to name a Data Bundle.
+      size:
+        type: string
+        format: int64
+        description: |-
+          The computed cumulated size of all Data Objects in bytes.
       created:
         type: string
         format: date-time
-        description: |-
+        description: >-
           Timestamp of object creation in RFC3339.
       updated:
         type: string
         format: date-time
-        description: |-
-          Timestamp of update in RFC3339, identical to create timestamp in systems
-          that do not support updates.
+        description: >-
+          Timestamp of update in RFC3339, identical to create timestamp in
+          systems that do not support updates.
       version:
         type: string
-        description: |-
-          A string representing a version, some systems may use checksum, a RFC3339
-          timestamp, or incrementing version number. For systems that do not support
-          versioning please use your update timestamp as your version.
+        description: >-
+          A string representing a version, some systems may use checksum, a
+          RFC3339 timestamp, or incrementing version number. For systems that do not
+          support versioning please use your update timestamp as your version.
+      mime-type:
+        type: string
+        description: >-
+          A string providing the mime-type of the Data Bundle.
+          For example, "application/json".
       checksums:
         type: array
+        description: >-
```

Comment:
Why is this an array here, but a single value for Object? That seems inconsistent.
I think it should be a simple reference to the `Checksum` with the computation method as described below.

Reply:
@ddietterich A Bundle or Object can have an array of checksums calculated using any hashing algorithm, not just MD5. The Checksum object provides the value and type of algorithm used.

Reply:
@ddietterich I think it's an array for `Object` also? (I agree with you they should match; I think they already do.)